Bugzilla.pm

```diff
@@ -761,10 +761,9 @@ sub check_rate_limit {
 }
 
 sub markdown_parser {
```

Comment:
looks like this should be renamed 'markdown', per the test that is failing:
`Can't locate object method "markdown" via package "Bugzilla" at t/markdown.t line 21.`